sympy/solvers/tests/test_diophantine.py

```diff
@@ -583,6 +582,58 @@ def check_solutions(eq):
 
     return okay
 
+
+def check_linear(eq, param=symbols("t", Integer=True)):
```

Comment:
I disagree.  The original tests only checked that the output matched the test case.  Which meant that when I changed the output's formatting, my valid solutions failed the test cases.  I suspect they were previously written by copy/pasting the output from the function into the test file, which would defeat the purpose of testing.